.ci/set_env.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+BRTG=$1
+BUILD_ID=$2
+
+ST_FILE="/etc/xlio_pkg/xlio/pkg_status"
+
+echo "1" > $ST_FILE
+
+apt-get update
+/usr/bin/apt install -y autoconf libtool debhelper build-essential ssh sudo
+
+cd /etc/xlio_pkg/xlio
+rm -rf libxlio/
+
+git clone https://github.com/Mellanox/libxlio.git 
+
+cd libxlio/
+git checkout $BRTG
+
+`QA_RPATHS=0x0002 PRJ_RELEASE=${BUILD_ID} contrib/build_pkg.sh -s -b -a "configure_options=--with-dpcp --disable-nginx --disable-utls" &`
+
```

Comment:
there is `-o` option of `build_pkg.sh` that point to output destination (by default: $PWD/pkg/packages/). You can use it.

Reply:
Unfortunately we can't use this option now. by default build_pkg.sh script clean all previously compiled packages. will discuss latter